File: src/main/java/fr/lewon/web/bot/entities/shop/Item.java
package fr.lewon.web.bot.entities.shop;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Item {

	@JsonProperty("id_item")
	private String id;
	
	@JsonProperty
	private String type;
	
	@JsonProperty
	private Integer price;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getType() {
		return type;
	}

	public void setType(String type) {
		this.type = type;
	}

	public Integer getPrice() {
		return price;
	}

	public void setPrice(Integer price) {
		this.price = price;
	}
	
}

File: src/main/kotlin/fr/lewon/bot/hh/entities/shop/Item.kt
package fr.lewon.bot.hh.entities.shop

import com.fasterxml.jackson.annotation.JsonProperty

class Item {
    @JsonProperty("id_item")
    var id: String? = null
    @JsonProperty
    var type: String? = null
    @JsonProperty
    var price: Int? = null

}